packages/core/src/sass/app.scss

```diff
@@ -10,6 +10,7 @@
 
 // General
 @import './details-form';
+@import './real-account-signup';
```

Comment:
is it wise to import it like this in core? shouldn't it be imported within the component itself?